DAOS-Support.md

```diff
@@ -92,7 +90,7 @@ $ mpirun -np 3 dcp -v /tmp/$USER/conts/p1cont1 /tmp/$USER/d
 [2020-04-23T17:17:51]   Files: 3
 [2020-04-23T17:17:51]   Links: 0
 ```
-## DAOS non-POSIX Data Movement Examples With dsync
+## DAOS non-POSIX Data Movement Examples With dcp
```

Comment:
@daltonbohning the description on example 4 I know isn't part of this PR, but it says the destination path is the relative path in the container, when it should really be the source. Might be easier to just add to this PR.

Reply:
@dsikich I agree; added.